scripts/build.sh

```diff
@@ -41,55 +38,27 @@ rm -f bin/*
 rm -rf pkg/*
 mkdir -p bin/
 
-# If its dev mode, only build for our self
-if [ "${VAULT_DEV_BUILD}x" != "x" ]; then
-    XC_OS=$(go env GOOS)
-    XC_ARCH=$(go env GOARCH)
-    XC_OSARCH=$(go env GOOS)/$(go env GOARCH)
-fi
-
-
-# The main method we need for building is in the cmd directory
-cd "${DIR}/cmd/${TOOL}"
-
 # Build!
 echo "==> Building..."
-gox \
-    -osarch="${XC_OSARCH}" \
+echo "Running:"
+echo "${GO_CMD} build -gcflags ${GCFLAGS} -ldflags -X github.com/hashicorp/${TOOL}/version.GitCommit=${GIT_COMMIT}${GIT_DIRTY} -o bin/${TOOL} -tags ${BUILD_TAGS} ."
+echo ""
+${GO_CMD} build \
+    -gcflags "${GCFLAGS}" \
     -ldflags "-X github.com/hashicorp/${TOOL}/version.GitCommit='${GIT_COMMIT}${GIT_DIRTY}'" \
-    -output "${DIR}/pkg/{{.OS}}_{{.Arch}}/${TOOL}" \
-    -tags="${BUILD_TAGS}" \
+    -o "bin/${TOOL}" \
+    -tags "${BUILD_TAGS}" \
```

Comment:
If you really want to echo the command being run you could do `set -x` and then `set +x`:

```go
set -x
${GO_CMD} build \
    -gcflags "${GCFLAGS}" \
    -ldflags "-X github.com/hashicorp/${TOOL}/version.GitCommit='${GIT_COMMIT}${GIT_DIRTY}'" \
    -o "bin/${TOOL}" \
    -tags "${BUILD_TAGS}" \
    .
set +x
```

That way you wouldn't have to update the `echo` command everytime the `go build` is updated.